modules/distribution/src/main/config/fluo-env.sh

```diff
@@ -66,10 +66,10 @@ setupClasspathFromSystem()
 
 
 # This function obtains Accumulo, Hadoop, and Zookeeper jars from
-# $FLUO_HOME/lib/deps/. Before using this function, go to that directoy and
-# follow the readme to download the dependencies.
+# $FLUO_HOME/lib/ahz/. Before using this function, make sure you run
```

Comment:
Seems like something should be added to fluo-env.sh about calling `fetch.sh extra`.